.circleci/config.yml

```
@@ -291,6 +291,12 @@ jobs:
     working_directory: ~/web3-onboard-monorepo/packages/gas
     steps:
       - node-build-steps
+  build-hwcommon:
```

```suggestion
  build-hw-common:
```
hyphen to follow naming convention